draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1710,9 +1710,15 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
    can be safely merged.
 </t>
 <t>
-   A client can send a <x:ref>TE</x:ref> header field indicating "trailers"
-   is acceptable, as described in <xref target="header.te"/>, to inform the
-   server that it will not discard trailer fields.
+   The presence of the keyword "trailers" in the TE header field (<xref
+   target="header.te"/>) indicates that the client is willing to accept
+   trailer fields, on behalf of itself and any downstream clients. For
+   requests from an intermediary, this implies that either: (a) all
```

Comment:
no (b)?

Reply:
Ah, yes. In the new world, (b) isn't allowed. Will fix, thx.